Documentation/network/clustermesh/gke-clustermesh-prep.rst

```diff
@@ -0,0 +1,128 @@
+.. _gs_clustermesh_gke_prep:
+
+**********************************
+GKE-to-GKE Clustermesh Preparation
+**********************************
+
+This is a step-by-step guide on how to install and prepare 
+Google Kubernetes Engine (GKE) clusters to meet the requirements 
+for the clustermesh feature.
+
+In this guide we will deploy two zonal, single node GKE clusters
+in different regions for the express purpose of creating a
+cost-effective environment to deploy a clustermesh to. Ideal for
+development/learning purposes.
+
+.. note::
+
+        The steps below require the `gcloud <https://cloud.google.com/sdk/docs/install>`__ CLI tool
+
+Create VPC
+##########
+
+1.  Create a VPC network in your GCP project. Environment variables are recommended as their
+    values will be references in later steps.
+
+    .. code-block:: bash
+
+        #  feel free to choose your own VPC network name
+        export PROJECT_ID="[GCP_PROJECT_ID]"
+        export VPC_NETWORK="my-gke-network"
+
+        gcloud compute networks create ${VPC_NETWORK} \
+          --subnet-mode=auto \
+          --project ${PROJECT_ID}
+
+        gcloud compute firewall-rules create ${VPC_NETWORK}-allow-internal \
+          --network ${VPC_NETWORK} \
+          --allow tcp,udp,icmp \
+          --source-ranges "10.128.0.0/9"
+
+
+Deploy clusters
+###############
+
+1.  We will once again set some environment variables for values that will be reused in 
+    later steps.
+
+    .. code-block:: bash
+
+        #  us-west1-a can be changed to any available location (`gcloud compute zones list`)
+        export CLUSTER="gke-1"
+        export ZONE="us-west1-a"
+        export POD_CIDR="10.0.0.0/18"
+        export SERVICES_CIDR="10.1.0.0/20"
+
+    Below is an example to deploy one GKE cluster. To create more clusters, follow the
+    steps again, using distinct cluster names, zones, pod CIDRs, and services CIDRs.
+
+    .. note::
+
+        During creation we also request to use ``"10.0.0.0/18"`` as the cluster (pod) CIDR
+        and ``"10.1.0.0/20"`` as the services CIDR. You may select change the CIDRs provided
+        they do not overlap with the pod and services CIDRs in your other cluster(s).
```

Comment:
```suggestion
You can use different pod and services CIDRs than in the example, but make sure that they are within the VPC range and that they do not overlap with the pod and services CIDRs in your other cluster(s).
```
Or something similar? The other paragraph already mentioned that the user should use different CIDRs for different clusters, so repeating the values here seems unnecessary.

Also, I think this note would be better placed right after the envvars, before the previous paragraph.

Reply:
Done.  As pods and services use an alias/secondary range that is separate from that of the VPC's, I just said that it needs to meet the IP address range rules and threw in a link to the Google docs.